scripts/build_translation_templates.py

```diff
@@ -0,0 +1,10 @@
+from scripts.test_translation_templates import (
+    build_schema_templates,
+    build_static_template,
+)
+
+if __name__ == '__main__':
+    print("--- Build Translation Templates ---")  # noqa: T001
+
+    build_static_template('app/translations/messages.pot')
+    build_schema_templates("app/translations/")
```

Comment:
Any reason not to have one script that does the extraction, that has something like a `--check`  command line argument for the testing?